cache/src/main/kotlin/com/dropbox/android/external/cache4/RealCache.kt

```diff
@@ -91,31 +92,29 @@ internal class RealCache<Key : Any, Value : Any>(
     }
 
     override fun get(key: Key): Value? {
-        val nowNanos = clock.currentTimeNanos
```

Comment:
I'm not sure it's wise to remove the time capture. by doing this you guarantee you'll expire all the entires based on the same time. with the new code you'll be using a later (changing) time to preform the check which could lead to inconsistent eviction results.

Reply:
hmm I'm now sure how to retain the old behavior as `kotlin.time` doesn't have an API for recording the absolute time or comparing `TimeMark`.

The goal was to move away from `System.nanoTime()` which uses `java.lang`. But since Kotlin native has [getTimeNanos](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.system/get-time-nanos.html) can we just use the existing custom `Clock` implementation and do `actual` / `expect` for KMP?